### ISSUE TITLE
Mouse doesn´t respond when using VNC on Xfce (raspberry)

### ISSUE THREAD

**Issue by lemi00** [patrikolausson/dispmanx_vnc#6]
Hi
I normally use the dispmanx-vnc to remotely control my graphic environment Xfce from raspberry.
However, since a few days ago I am unable to do so.
The service starts, I can see the remote computer monitor but I can't control it using the mouse or keyboard. The cursor simply don't move.
Regards

**Comment by patrikolausson** [comment]
Did you try the following from the readme before starting dispmanx_vncserver: sudo modprobe evdev
Did you run the server with sudo?
Does it work if you have a keyboard and/or mouse connected to the pi?
Did you make sure that the vnc client isn't in a "view only mode" or something like that?

**Comment by lemi00** [comment]
First of all, thanks a lot for your quick answer.
Just for the record I have been using, for quit a while dispmanx_vnc witrh
success. The problems are recent.
Now, regarding your suggestions:
I already tried the "sudo modprobe evdev" command
I always start the server with sudo "sudo systemctl start dispmanx-vnc"
The mouse and keyboard work just fine when connected directly to the pi and
Teamviewer also works.
I have checked and its not in view mode. This problem exists in all PC and
mobile phones that I have tried.

The problem only occurs in Xfce.
In emulationstation works fine (keyboard).
In Kody everything works also fine (mouse and keyboard).

I was told that recently they have changed the xorg server and had to
recompile the dispmanx, and I think that the problems start to occur since
then.

Thanks again for your help.
I look forward hearing from you again.

2017-01-19 20:20 GMT+00:00 patrikolausson <notifications@github.com>:
